src/opnsense/mvc/app/controllers/OPNsense/Unbound/DotController.php

```diff
@@ -32,8 +32,9 @@
 
 class DotController extends IndexController
 {
-    public function indexAction()
+    public function indexAction($selected = null)
```

Comment:
looking at this again, it might be cleaner to use two controllers `DotController.php` and `ForwardController.php` so the url's are a bit more logical (`/ui/unbound/dot/` and `/ui/unbound/forward/` ads you can omit the index part)

Reply:
Makes sense, trying to keep the change as minimal as possible was the reason I did it this way with this drawback as a result.

Reply:
see https://github.com/opnsense/core/pull/5606/commits/2b63edd076871569671591707debc97d47c96c44